x-pack/plugin/core/src/main/resources/logs-mappings.json

```diff
@@ -1,14 +1,312 @@
 {
   "template": {
     "mappings": {
+      "dynamic_templates": [
+        {
+          "match_ip": {
+            "match_mapping_type": "string",
+            "match": "ip",
+            "runtime": {
+              "type": "ip"
+            }
+          }
+        },
+        {
+          "message": {
+            "path_match": "message",
+            "mapping": {
+              "type": "match_only_text"
+            }
+          }
+        },
+        {
+          "log_level": {
+            "path_match": "log.level",
+            "mapping": {
+              "type": "keyword",
+              "ignore_above": 1024
+            }
+          }
+        },
+        {
+          "log_logger": {
```

Comment:
Do we need this indexed by default? Runtime field instead?

Reply:
It's not used as often as the `log.level`, but it is useful to be able to filter for logs of a specific sub-section of the app. I think it's debatable. But it's also not set by all logger frameworks. So if it's set, why not index it? Not feeling strongly about this field, though.